modules/ROOT/pages/ch-deploy-mule-application-mmp-task.adoc

```diff
@@ -8,12 +8,16 @@ endif::[]
 To successfully deploy your Mule application to CloudHub, consider the following requirements:
 
 * The host and port number of your HTTP Listener flow sources are properly configured. +
- If you are using the HTTP Listener as sources for your flow, you need to set its host to *0.0.0.0* and its port to *${http.port}*. +
++
+If you are using the HTTP Listener as sources for your flow, you need to set its host to *0.0.0.0* and its port to *${http.port}*.
++
 CloudHub then dynamically allocates a port at deployment time.
-* All your external classes and resources are properly declared in the `mule-artifact.json` file of your Mule application. +
- Due to Mule 4.x classloading isolation mechanism, all external classes and resources must be explicitly declared in the "exportedPackages" and "exportedResources" fields on the mule-artifact.json file before packaging and deploying the application. +
 +
-See the xref:how-to-export-resources.adoc[How to Export Resources] task to learn more about working with external resources in your Mule application.
+* All your external classes and resources are properly declared in the `mule-artifact.json` file of your Mule application.
++
+Due to Mule 4.x classloading isolation mechanism, all external classes and resources must be explicitly declared in the "exportedPackages" and "exportedResources" fields on the mule-artifact.json file before packaging and deploying the application.
```

Comment:
`mule-artifact.json` ?
In most cases, you are changing Mule application to Mule app. Not sure if it matters but here it is Mule application.